zipkin-storage/cassandra/src/main/java/zipkin/storage/cassandra/Schema.java

```diff
@@ -78,10 +76,12 @@ private static KeyspaceMetadata getKeyspaceMetadata(String keyspace, Cluster clu
   }
 
   static void ensureExists(String keyspace, Session session) {
-    KeyspaceMetadata keyspaceMetadata = getKeyspaceMetadata(keyspace, session.getCluster());
```

Comment:
this was the chicken-egg. if the keyspace wasn't installed, getKeyspaceMetadata would throw